app/src/main/res/layout/terms_and_conditions.xml

```diff
@@ -1,72 +1,193 @@
+                android:layout_height="wrap_content"
+                android:breakStrategy="high_quality"
+                android:paddingLeft="3dp"
+                android:paddingRight="3dp"
+                android:text="@string/tac_data_collection"
+                app:layout_constraintBottom_toTopOf="@+id/tac_contactInformation"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/tac_dataCollected" />
+
+            <TextView
+                android:id="@+id/tac_contactInformation"
+                android:layout_width="match_parent"
+                android:layout_height="31dp"
+                android:layout_marginBottom="1dp"
+                android:fontFamily="sans-serif-medium"
+                android:paddingLeft="3dp"
+                android:paddingRight="3dp"
+                android:text="Contact Information"
+                android:textAppearance="@style/TextAppearance.AppCompat.Medium"
+                android:textSize="24sp"
+                app:layout_constraintBottom_toTopOf="@+id/tac_contactInformation_text"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintHorizontal_bias="1.0"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/tac_dataCollected_text" />
+
+            <TextView
+                android:id="@+id/tac_contactInformation_text"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:autoLink="all"
+                android:linksClickable="true"
+                android:paddingLeft="3dp"
+                android:paddingRight="3dp"
+                android:text="@string/tac_contact_information"
+                app:layout_constraintBottom_toTopOf="@+id/tac_aboutApp"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/tac_contactInformation" />
+
+            <TextView
+                android:id="@+id/tac_aboutApp"
+                android:layout_width="match_parent"
+                android:layout_height="31dp"
+                android:layout_marginBottom="1dp"
+                android:fontFamily="sans-serif-medium"
+                android:paddingLeft="3dp"
+                android:paddingRight="3dp"
+                android:text="About the app"
+                android:textAppearance="@style/TextAppearance.AppCompat.Medium"
+                android:textSize="24sp"
+                app:layout_constraintBottom_toTopOf="@+id/tac_aboutApp_text"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/tac_contactInformation_text" />
+
+            <TextView
+                android:id="@+id/tac_aboutApp_text"
```

Comment:
Dit gedeelte is wel heel technisch. Moet iedere gebruiker dit kunnen begrijpen? In niet-technische termen staat het eigenlijk elders ook al...

Reply:
Ja misschien kan het wel wat minder technisch, maar denk wel dat het fijn is voor appgebruikers om te weten wat er gebeurt als ze iets wel of niet aanvinken. Soort handleiding. maar weet ook niet of dit de plek er voor is

Reply:
Misschien wat algemene informatie over wat er uiteindelijk mee gebeurt? Zoals een linkje naar eerdere publicaties?

Reply:
Linkje toegevoegd, en de gebruikerstekst iets minder technisch gemaakt